x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/inference/ltr/LearningToRankServiceTests.java

```diff
@@ -77,6 +79,23 @@ public class LearningToRankServiceTests extends ESTestCase {
         .setModelType(TrainedModelType.TREE_ENSEMBLE)
         .setInferenceConfig(new RegressionConfig(null, null))
         .build();
+    public static final TrainedModelConfig MODEL_WITH_DUPS_CONFIG = TrainedModelConfig.builder()
```

Comment:
Just a nit but this is probably better written using a copy of `GOOD_MODEL_CONFIG` and modifying only what is changed:

```java
public static final TrainedModelConfig MODEL_WITH_DUPS_CONFIG = new TrainedModelConfig.Builder(GOOD_MODEL_CONFIG)
  .setModelId(MODEL_WITH_DUPS)
  .setInput(new TrainedModelInput(List.of("field1", "field2", "feature_1", "feature_2")))
  .build()
```